clouddrift/sphere.py

```diff
@@ -195,20 +195,23 @@ def recast_lon(lon: np.ndarray, lon0: Optional[float] = -180) -> np.ndarray:
     Returns
     -------
     np.ndarray or float
-        Converted longitudes in the range `[lon0, lon0+360]`
+        Converted longitudes in the range `[lon0, lon0+360[`
```

Comment:
Should it not be `[lon0, lon0+360)` ?

Reply:
Both seem accepted https://en.wikipedia.org/wiki/ISO_31-11

Reply:
Interesting, I didn't know.